static/css/home.css

```diff
@@ -36,10 +36,6 @@
         float: left;
     }
 
-    .nav-masthead {
-        float: right;
-    }
-}
```

Comment:
Unfortunately you deleted too much. :) The last `}` should stay, it closes `@media.`